### DISCUSSION TITLE
What Do "REL/DBG" in the Program Filenames Stand For?

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [dakanji/RefindPlus#162] [General]
Can anyone explain the meaning of DBG/REL used in the efi file name of this tool?

![image](https://github.com/dakanji/RefindPlus/assets/11155854/46c34b19-d8c7-4b22-9da3-50963a8e91a7)

**Comment by dakanji** [comment]
These are fairly common abbreviations for build types:
REL = RELEASE
DBG = DEBUG

Quick web search:

https://learn.microsoft.com/en-us/vcpkg/users/examples/overlay-triplets-linux-dynamic
```
-- Configuring x64-linux-dynamic-dbg
-- Configuring x64-linux-dynamic-rel
``` 

https://wiki.hyperledger.org/pages/viewpage.action?pageId=41584046
```
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Building x64-linux-dbg
```

https://firefox-source-docs.mozilla.org/setup/configuring_build_options.html
```
env MOZCONFIG=/path/to/mozconfig-dbg ./mach build
env MOZCONFIG=/path/to/mozconfig-rel-opt ./mach build
```
